mypyc/irbuild/mapper.py

```diff
@@ -117,6 +118,8 @@ def type_to_rtype(self, typ: Type | None) -> RType:
             return none_rprimitive
         elif isinstance(typ, UnionType):
             return RUnion.make_simplified_union([self.type_to_rtype(item) for item in typ.items])
+        elif isinstance(typ, IntersectionType):
+            return object_rprimitive
         elif isinstance(typ, AnyType):
             return object_rprimitive
         elif isinstance(typ, TypeType):
```

Comment:
```suggestion
        elif isinstance(typ, IntersectionType | AnyType | TypeType):
```

Reply:
Don't want to do that as it's upstream code , but additionally, it creates more work if any of these entries needs to be specialized.